package.json

```diff
@@ -81,5 +84,19 @@
     "plugins": [
       "transform-flow-strip-types"
     ]
+  },
+  "lint-staged": {
+    "src/*.js": [
+      "prettier --trailing-comma all --write \"src/*.js\"",
+      "git add"
+    ],
+    "scripts/*.js": [
+      "prettier --trailing-comma all --write \"scripts/*.js\"",
+      "git add"
+    ],
+    "test/*.js": [
```

Comment:
@existentialism `test/*.js` will cover just files inside script dir, but not below test directory, so `fixtures`/`debug-fixtures` shouldn't be affected.